unstructured/ingest/connector/salesforce.py

```diff
@@ -184,10 +185,12 @@ def _get_response(self):
     def get_record(self) -> OrderedDict:
         # Get record from Salesforce based on id
         response = self._get_response()
-        logger.debug(f"response from salesforce record request: {response}")
+        logger.debug(f"response from salesforce record request: {json.dumps(response)}")
         records = response["records"]
         if not records:
-            raise ValueError(f"No record found with record id {self.record_id}: {response}")
+            raise ValueError(
+                f"No record found with record id {self.record_id}: {json.dumps(response)}"
+            )
         record_json = records[0]
```

Comment:
do we really want to be logging these records at all? ...that aside, they could have sensitive data in them. probably best to just log the record id?

Reply:
@ryannikolaidis I was thinking about that. But it is Debug. I will change to just record_id.